code-coverage.cmake

```diff
@@ -456,6 +460,11 @@ function(target_code_coverage TARGET_NAME)
         endif()
         add_dependencies(ccov ccov-${target_code_coverage_COVERAGE_TARGET_NAME})
 
+        if(NOT TARGET ccov-html)
+          add_custom_target(ccov-html)
+        endif()
+        add_dependencies(ccov-html ccov-html-${target_code_coverage_COVERAGE_TARGET_NAME})
+
```

Comment:
ccov-html/ccov-html-<TARGET> is only available for GCC with this change. But because it's here, it's being added for both GCC and Clang. Can this be shifted down into the if(GNU) section just underneath?

Reply:
Oh, I missed that, will fix it.